target/linux/lantiq/image/Makefile

```diff
@@ -757,89 +823,113 @@ endif
 ifeq ($(SUBTARGET),falcon)
 
 define Device/lantiq_easy98000-nor
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000
+  DEVICE_VARIANT := Falcon Eval Board NOR
   IMAGE_SIZE := 3904k
   DEVICE_DTS := EASY98000NOR
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board NOR
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-nor
 
 define Device/lantiq_easy98000-nand
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000
+  DEVICE_VARIANT := Falcon Eval Board NAND
   IMAGE_SIZE := 3904k
   DEVICE_DTS := EASY98000NAND
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board NAND
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-nand
 
 define Device/lantiq_easy98000-sflash
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000
+  DEVICE_VARIANT := Falcon Eval Board SFLASH
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY98000SFLASH
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board SFLASH
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-sflash
 
 define Device/lantiq_falcon-mdu
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := MDU
+  DEVICE_VARIANT := Lantiq Falcon / VINAXdp MDU Board
   IMAGE_SIZE := 7424k
   DEVICE_DTS := FALCON-MDU
-  DEVICE_TITLE := MDU - Lantiq Falcon / VINAXdp MDU Board
 endef
 TARGET_DEVICES += lantiq_falcon-mdu
 
 define Device/lantiq_easy88388
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY88388
+  DEVICE_VARIANT := Falcon FTTDP8 Reference Board
```

Comment:
Same here, don't use VARIANT if there is only one variant, this is meant for things like `v1` vs. `v2` or `8M` vs. `16M` or `NAND` vs. `NOR` in case both variants exist (though not necessarily more than one of them is supported by OpenWrt)